src/helper/UserSession.cpp

```diff
@@ -82,6 +82,8 @@ namespace SDDM {
             const QString cmd = QStringLiteral("%1 %2").arg(mainConfig.Wayland.SessionCommand.get()).arg(m_path);
             qInfo() << "Starting Wayland user session:" << cmd;
             m_process->start(mainConfig.Wayland.SessionCommand.get(), QStringList{m_path});
+            m_process->closeWriteChannel();
+            m_process->closeReadChannel(QProcess::StandardOutput);
```

Comment:
Maybe it would make sense to only close the standard output if it's not a greeter? For greeters I find it's useful to see what's going on.